src/types/sdk/services/proxy.ts

```diff
@@ -24,6 +24,7 @@ type BasisTheoryQueryParams = {
 type ProxyQuery = Partial<BasisTheoryQueryParams>;
 
 interface ProxyRequestOptions extends RequestOptions {
+  includeResponseHeaders?: boolean;
```

Comment:
what's the purpose of this option? would always returning the response headers potentially be a breaking change?

Reply:
Did not want to just add `headers` to all responses bc it's an internal-ish thing for now.